vendor/sigs.k8s.io/apiserver-network-proxy/konnectivity-client/pkg/client/client.go

```diff
@@ -134,7 +124,6 @@ func (t *grpcTunnel) serve(c *grpc.ClientConn) {
 				t.connsLock.Lock()
 				delete(t.conns, resp.ConnectID)
 				t.connsLock.Unlock()
-				return
```

Comment:
I think we should keep the `return` here, otherwise we leaks the `serve` goroutines.